src/Sylius/Component/Core/Repository/OrderRepositoryInterface.php

```diff
@@ -54,11 +54,12 @@ public function countByCustomerAndPaymentState(CustomerInterface $customer, $sta
      * Gets revenue group by date
      * between particular dates
      *
-     * @param array $configuration
+     * @param array  $configuration
+     * @param string $baseCurrency
      *
      * @return array
      */
-    public function revenueBetweenDatesGroupByDate(array $configuration = array());
+    public function revenueBetweenDatesGroupByDate(array $configuration = array(), $baseCurrency);
```

Comment:
Here, the first argument `$configuration` has a default value, but the next argument `$baseCurrency` does not.